x-pack/test/security_solution_cypress/cypress/cypress_ci_serverless_qa.config.ts

```diff
@@ -16,6 +16,7 @@ export default defineCypressConfig({
   },
   defaultCommandTimeout: 150000,
   env: {
+    CLOUD_SERVERLESS: true,
```

Comment:
This value is already set in our test code, more specifically in the scripts we call in our `package.json`. Having it here as well may lead to misunderstandings or wrong values, we must have it in just one place. Personally, I would prefer to have it outside this configuration file.